docs/src/content/docs/fr/showcase.md

```diff
@@ -1,20 +1,41 @@
 ---
 title: Starlight Showcase
-description: Voir les sites construits avec Starlight sur le web !
+description: Découvrez les sites construits avec Starlight et les outils communautaires qui étendent Starlight !
 ---
 
-Starlight est déjà utilisé en production sur le web. Voyez quelques exemples ci-dessous, et publiez votre propre lien si vous avez construit votre site de documentation avec Starlight !
+:::tip[Ajoutez le vôtre !]
+Avez-vous construit un site Starlight ou un outil pour Starlight ?
+Ouvrez une PR en ajoutant un lien vers cette page !
+:::
+
+## Sites
+
+Starlight is already being used in production. These are some of the sites around the web:
 
 - [PubIndexAPI Docs](https://docs.pubindexapi.com/)
 - [har.fyi](https://har.fyi/)
 - [QBCore Docs](https://brycerussell.github.io/qbcore-docs/)
 - [Astro Error Pages](https://astro-error-page-documentation.vercel.app/)
 - [Felicity](https://felicity.pages.dev/)
 - [NgxEditor](https://sibiraj-s.github.io/ngx-editor/)
+- [pls](https://dhruvkb.github.io/pls)
 - [Terrateam Docs](https://terrateam.io/docs)
 - [xs-dev docs](https://xs-dev.js.org)
 - [Athena OS](https://www.athenaos.org)
 - [simple-fm](https://simple.arciniega.one)
-- [Starwoof](https://starwoof.vercel.app/) - Regardez Ben et Chris d'Astro construire ce site Starlight présentant les animaux de compagnie d'Astro en temps réel ! [Twitch stream: Astro x Starlight - BLAZING FAST ECO-FRIENDLY NON-GMO ORGANIC docs](https://www.twitch.tv/videos/1841159960)
 
 Voir [les dépôts de projets publics utilisant Starlight sur GitHub](https://github.com/withastro/starlight/network/dependents).
+
+## Plugins de la communauté
+
+Ces outils communautaires, plugins et intégrations fonctionnent avec Starlight pour étendre ses fonctionnalités.
```

Comment:
In the English “community tools, plugins, and integrations”, “community” applies to all three nouns in the list, so perhaps we need to tweak this?

```suggestion
Ces outils, plugins et intégrations communautaires fonctionnent avec Starlight pour étendre ses fonctionnalités.
```